reference/carts.sf.yml

```diff
@@ -250,6 +250,43 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/deleteCartsItems'
+  '/cart/{cartId}/currency':
+    post:
+      tags:
+        - Cart Currency
+      summary: Update Cart currency
+      description: |-
+        Update currency of the *Cart*.
```

Comment:
Maybe we can add some notes around the fact that any promotions, gift certificate applied to the cart...will may get removed from the cart.

I think we send this message in Cornerstone for instance
![image](https://user-images.githubusercontent.com/43359039/161454777-2302ef3f-3c3b-4cce-8ae7-489c1f91fe04.png)

Reply:
Thank you for suggestion. Yes, we should add those description.